project/context.go

```diff
@@ -38,6 +41,7 @@ type Context struct {
 	MountPath     string
 	RelDockerPath string
 	BuilderTag    string
+	AppSource     appsource.AppSource
```

Comment:
Instead of adding AppSource to Context, we should create a local instance of it where it's needed instead

Reply:
done